src/containerapp/azext_containerapp/custom.py

```diff
@@ -175,17 +211,288 @@ def update_containerapp_yaml(cmd, name, resource_group_name, file_name, from_rev
         handle_raw_exception(e)
 
 
-def create_containerapp_yaml(cmd, name, resource_group_name, file_name, no_wait=False):
-    yaml_containerapp = process_loaded_yaml(load_yaml_file(file_name))
+def _override_parameters_into_containerapp(cmd,
+                                           containerapp_def,
+                                           image=None,
+                                           container_name=None,
+                                           managed_env=None,
+                                           min_replicas=None,
+                                           max_replicas=None,
+                                           target_port=None,
+                                           transport=None,
+                                           ingress=None,
+                                           revisions_mode=None,
+                                           secrets=None,
+                                           set_env_vars=None,
+                                           remove_env_vars=None,
+                                           replace_env_vars=None,
+                                           remove_all_env_vars=False,
+                                           cpu=None,
+                                           memory=None,
+                                           registry_server=None,
+                                           registry_user=None,
+                                           registry_pass=None,
+                                           dapr_enabled=None,
+                                           dapr_app_port=None,
+                                           dapr_app_id=None,
+                                           dapr_app_protocol=None,
+                                           revision_suffix=None,
+                                           startup_command=None,
+                                           args=None,
+                                           tags=None):
+    # Containers
+    container_needs_update = (image is not None or container_name is not None or set_env_vars is not None or
```

Comment:
Same here add what is this for? especially if we are adding special handling on client side which is different from how the API behaves (making the behavior inconsistent from other clients). Thanks!

Reply:
Removed yaml for this release